### DISCUSSION TITLE
About

### DISCUSSION THREAD

**Discussion by giscus** [SkyLeite/blog#3] [Announcements]
# About

https://leite.dev/pages/test-page/

**Comment by jmone** [comment]
great engineer, come on!

I'm 34 years old, and a programmer from China.

**Reply by SkyLeite** [reply]
Thank you! That's very flattering :)

**Comment by Youbuu2** [comment]
Thank you for your work on Golden Fantasia. As a 24-year-old recent Computer Science graduate, I’m not yet familiar with reverse engineering or anything of that nature, but I’m currently focused on building my knowledge in embedded systems and pursuing a career in that field. Reading your blog has given me a renewed sense of hope and determination, and I deeply appreciate the insight and inspiration it provides.

**Reply by SkyLeite** [reply]
Thank you so much for the kind words! I'm flattered to know that I managed to inspire your journey a little bit. Hopefully one day I'll be learning from embedded systems from you :)
